### PR TITLE
fix: (CXSPA-1109) - Styles of links for cart table items

### PR DESCRIPTION
**Pull request by sdrozdsap**
The same styles are shared between quick order, cart and wishlist tables therefore I've fixed this issue for all of those occurrences:
image

image
image

### CODE REVIEW

File: feature-libs/cart/base/styles/components/_cart-item-list.scss [hunk 1]
```diff
@@ -62,8 +62,8 @@
   }
 
   .cx-link {
-    text-decoration: none;
-    color: var(--cx-color-text);
+    text-decoration: underline;
```
Comment (Zeyber) [inline feature-libs/cart/base/styles/components/_cart-item-list.scss]: Here we are changing existing css properties which is a breaking change. @Pio-Bar has a few PRs where he has handled this issue. [Here is one example](https://github.com/SAP/spartacus/pull/18603/files).

tldr; Here's what we need to do:


- We add our new properties wrapped in a feature toggle to overwrite old properties along with cleanup ticket. 

Example:
```
    // old properties

    // TODO: (CXSPA-XXXX) - Remove feature flag next major release
    @include forFeature('a11yFeature') {
      // new properties
    }
```

- We need to add `useFeatureStyles` to our component constructor as well to enable them. ie. `useFeatureStyles('a11yFeature');`.
- We need to add documentation about the new flag in `projects/core/src/features-config/feature-toggles/config/feature-toggles.ts`.
- We should add this flag to our feature toggle config in the `app.module.ts` to enable it by default.

**Review by Zeyber — CHANGES_REQUESTED** [review]

File: feature-libs/cart/base/styles/components/_cart-item-list.scss (hunk 1 above)
Comment (sdrozdsap) [reply feature-libs/cart/base/styles/components/_cart-item-list.scss]: I've applied these changes

**Review by Zeyber — APPROVED** [review]

**Review by developpeurweb — APPROVED** [review]
